packages/patternfly-4/react-core/src/demos/PaginationTable.md

```diff
@@ -50,17 +70,50 @@ class ComplexPaginationTableDemo extends React.Component {
   }
 
   render() {
-    const { loading } = this.state;
+    const { loading, res } = this.state;
+    if (res.length === 0) {
+      const rows = [{
+        heightAuto: true,
+        cells: [
+          {
+            props: { colSpan: 8 },
+            title: (
+            <Bullseye>
+              <EmptyState variant={EmptyStateVariant.small}>
+                <EmptyStateIcon icon={SearchIcon} />
+                <Title headingLevel="h2" size="lg">
+                  No results found
+                </Title>
+                <EmptyStateBody>
+                  No results match the filter criteria. Remove all filters or clear all filters to show results.
+                </EmptyStateBody>
+              </EmptyState>
+            </Bullseye>
+            )
+          },
+        ]
+      }]
+      
+      return (
+        <React.Fragment>
+          {this.renderPagination()}
+          <Table cells={['Title', 'Body']} rows={rows} aria-label="Pagination Table Demo">
+            <TableHeader />
+            <TableBody />
+          </Table>
+        </React.Fragment>
+      );
+    }
     return (
       <React.Fragment>
         {this.renderPagination()}
         {!loading && (
-          <Table cells={['Title', 'Body']} rows={this.state.res.map(post => [post.title, post.body])}>
+          <Table cells={['Title', 'Body']} rows={res.map(post => [post.title, post.body])} aria-label="Pagination Table Demo">
             <TableHeader />
             <TableBody />
           </Table>
         )}
-        {loading && <center><Title size="3xl">Please wait while loading data</Title></center>}
+        {loading && <center><Spinner size="xl"/></center>}
```

Comment:
I'm not seeing a spinner. I briefly see the empty state prior to loading the table rows. Is there a way to better see this?